config/development-cass-es.yaml

```
@@ -20,7 +20,7 @@ persistence:
           host: "127.0.0.1:9200"
         indices:
           visibility: temporal_visibility_v1_dev
-          # secondary_visibility: temporal_visibility_v2_dev
+          # secondary_visibility: temporal_visibility_v1_index_by_date
```

Let's use a more generic term.

```suggestion
          # secondary_visibility: temporal_visibility_v1_secondary
```